.github/workflows/ci_tests.yaml

```diff
@@ -54,36 +54,36 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.10', '3.12']
+        python-version: ['3.10', '3.13']
         os: [ubuntu-latest, macos-latest, windows-latest]
         # Is it a draft Pull Request (true or false)?
         isDraft:
           - ${{ github.event.pull_request.draft }}
-        # Only run two jobs (Ubuntu + Python 3.10/3.12) for draft PRs
+        # Only run two jobs (Ubuntu + Python 3.10/3.13) for draft PRs
         exclude:
           - os: macos-latest
             isDraft: true
           - os: windows-latest
             isDraft: true
         # Pair Python 3.10 with the minimum supported versions of NumPy, pandas, Xarray
-        # and Python 3.12 with the latest versions of NumPy, pandas, Xarray
-        # Only install optional packages on Python 3.12
+        # and Python 3.13 with the latest versions of NumPy, pandas, Xarray
+        # Only install optional packages on Python 3.13
         include:
           - python-version: '3.10'
             numpy-version: '1.24'
             pandas-version: '=2.0'
             xarray-version: '=2023.04'
             optional-packages: ''
-          - python-version: '3.12'
+          - python-version: '3.13'
             numpy-version: '2.1'
             pandas-version: ''
             xarray-version: ''
-            optional-packages: ' contextily geopandas ipython pyarrow rioxarray sphinx-gallery'
+            optional-packages: ' contextily geopandas ipython rioxarray sphinx-gallery'
```

Comment:
Since we're including PyArrow in the Python 3.11 CI build as of #3605, let's just remove PyArrow here until it supports Python 3.13 to not hold up #3606.

Reply:
We can't remove PyArrow in the Python 3.13 job, because we need both pandas<2.2+pyarrow (in Python 3.11) and pandas>=2.2+pyarrow (in Python 3.12 or 3.13) to test the behavior changes in pands 2.2.

Reply:
I was thinking we could do (after #3606) Python 3.11+pandas=2.1+pyarrow; Python 3.12+pandas=2.2+pyarrow, but anyways, this won't work yet because `mamba install` is still complaining about some incompatible depedencies (looks like something on `dvc`):

```
The following packages are incompatible
├─ dvc >=3.56.0  is installable and it requires
 |    ...
│  └─ scmrepo >=3.3.8,<4 , which requires
│     └─ asyncssh >=2.13.1,<3 , which requires
│        └─ python-gssapi >=1.2.0  with the potential options
│           ├─ python-gssapi [1.6.10|1.6.11|1.6.12|1.6.13|1.6.9] would require
│           │  └─ python_abi 3.6 *_pypy36_pp73, which can be installed (as previously explained);
│           ├─ python-gssapi [1.6.10|1.6.11|...|1.7.0] would require
│           │  └─ python >=3.6,<3.7.0a0 , which can be installed;
│           ├─ python-gssapi [1.6.10|1.6.11|...|1.8.1] would require
│           │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│           ├─ python-gssapi [1.6.10|1.6.11|...|1.8.3] would require
│           │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│           ├─ python-gssapi [1.6.10|1.6.11|...|1.9.0] would require
│           │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│           ├─ python-gssapi [1.6.12|1.6.13|...|1.7.3] would require
│           │  └─ python_abi 3.7 *_pypy37_pp73, which can be installed (as previously explained);
│           ├─ python-gssapi [1.7.2|1.7.3|...|1.9.0] would require
│           │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│           ├─ python-gssapi [1.7.3|1.8.0|1.8.1|1.8.2] would require
│           │  └─ python_abi 3.8 *_pypy38_pp73, which can be installed (as previously explained);
│           ├─ python-gssapi [1.7.3|1.8.0|1.8.1|1.8.2|1.8.3] would require
│           │  ├─ pypy3.9 >=7.3.13 , which can be installed (as previously explained);
│           │  └─ python_abi 3.9 *_pypy39_pp73, which can be installed (as previously explained);
│           ├─ python-gssapi [1.8.2|1.8.3|1.9.0] would require
│           │  └─ python >=3.11,<3.12.0a0 , which can be installed;
│           ├─ python-gssapi [1.8.2|1.8.3|1.9.0] would require
│           │  └─ python_abi 3.12.* *_cp312, which can be installed (as previously explained);
│           ├─ python-gssapi 1.8.2 would require
│           │  └─ pypy3.8 >=7.3.11 , which can be installed (as previously explained);
│           └─ python-gssapi 1.8.2 would require
│              └─ pypy3.9 >=7.3.11 , which can be installed (as previously explained);
└─ python 3.13**  is not installable because there are no viable options
   ├─ python 3.13.0 conflicts with any installable versions previously reported;
   └─ python [3.13.0rc1|3.13.0rc2|3.13.0rc3] would require
      └─ _python_rc, which does not exist (perhaps a missing channel).
```

Wait for https://github.com/conda-forge/python-gssapi-feedstock/pull/32.

Reply:
Also need to wait for https://github.com/conda-forge/kombu-feedstock/pull/78:

```
error    libmamba Could not solve for environment specs
    The following packages are incompatible
    ├─ dvc =* * is installable with the potential options
    ...
    │  └─ dvc [3.49.0|3.50.0|...|3.58.0] would require
    │     └─ celery =* * with the potential options
    ...
    │        ├─ celery [5.3.0|5.3.1|5.3.4] would require
    │        │  └─ kombu [>=5.3.0,<6.0 *|>=5.3.1,<6.0 *|>=5.3.2,<6.0 *] with the potential options
    │        │     ├─ kombu [5.3.0|5.3.1|...|5.4.1] would require
    │        │     │  └─ python >=3.10,<3.11.0a0 * with the potential options
    │        │     │     ├─ python [3.10.7|3.10.8|3.11.0|3.9.14|3.9.15], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.0|3.10.1|...|3.9.9], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.4|3.9.12], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.5|3.9.13], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.0|3.10.1|...|3.10.9], which can be installed;
    │        │     │     └─ python [3.10.6|3.11.0], which can be installed (as previously explained);
    │        │     ├─ kombu [5.3.0|5.3.1|...|5.4.1] would require
    │        │     │  └─ python >=3.11,<3.12.0a0 * with the potential options
    │        │     │     ├─ python [3.10.7|3.10.8|3.11.0|3.9.14|3.9.15], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.6|3.11.0], which can be installed (as previously explained);
    │        │     │     └─ python [3.11.0|3.11.1|...|3.11.9], which can be installed;
    │        │     ├─ kombu [5.3.0|5.3.1|5.3.2] would require
    │        │     │  └─ python_abi ==3.8 *_pypy38_pp73, which can be installed (as previously explained);
    │        │     ├─ kombu [4.6.10|4.6.11|...|5.4.0] would require
    │        │     │  └─ python >=3.8,<3.9.0a0 *, which can be installed;
    │        │     ├─ kombu [5.3.0|5.3.1|...|5.4.0] would require
    │        │     │  └─ python_abi ==3.9 *_pypy39_pp73, which can be installed (as previously explained);
    │        │     ├─ kombu [5.3.0|5.3.1|...|5.4.1] would require
    │        │     │  └─ python >=3.9,<3.10.0a0 * with the potential options
    │        │     │     ├─ python [3.10.7|3.10.8|3.11.0|3.9.14|3.9.15], which can be installed (as previously explained);
    │        │     │     ├─ python [3.9.10|3.9.12|...|3.9.9], which can be installed;
    │        │     │     ├─ python [3.9.0|3.9.1], which can be installed (as previously explained);
    │        │     │     ├─ python 3.9.1, which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.0|3.10.1|...|3.9.9], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.4|3.9.12], which can be installed (as previously explained);
    │        │     │     ├─ python [3.10.5|3.9.13], which can be installed (as previously explained);
    │        │     │     ├─ python 3.9.2, which can be installed (as previously explained);
    │        │     │     └─ python [3.9.4|3.9.5|3.9.6|3.9.7], which can be installed (as previously explained);
    │        │     └─ kombu [5.3.2|5.3.3|...|5.4.1] would require
    │        │        └─ python_abi =3.12 *_cp312, which can be installed (as previously explained);
```

Reply:
https://github.com/conda-forge/python-gssapi-feedstock/pull/32 has been superseded by https://github.com/conda-forge/python-gssapi-feedstock/pull/33. 

So we still need to wait for https://github.com/conda-forge/kombu-feedstock/pull/78. kombu's Python 3.13 support was added in https://github.com/celery/kombu/pull/2052, which first appeared in kombu v5.5.0rc2. 

kombu v5.5.0rc2 was released two months ago, and there is still no kombu v5.5.0 out. Not sure how long we have to wait.

Reply:
Ok, for reference, we applied a workaround for `dvc` in #3695, and `pyarrow` now has a Python 3.13 build since https://github.com/conda-forge/pyarrow-feedstock/pull/139.